### DISCUSSION TITLE
How to restore the TSDB in a new Prometheus instance using the snapshot from an older Prometheus instance?

### DISCUSSION THREAD

**Discussion by Ezhil2k** [prometheus/prometheus#13311] [Q&A]
I have created a snapshot from an older instance which is running on Kubernetes. The snapshot includes the following components:

- `chunks/`
- `meta.json`
- `tombstones`
- `index`

In the new instance, after enabling the admin API, I replaced the contents from the snapshot into the new instance in the same directory. 

However, I'm encountering an issue. The time series data from the snapshot is not being displayed. Instead, it just continues to show the current time series data. I followed the steps provided in this article: [How to Backup and Restore Prometheus](https://suraj.io/post/how-to-backup-and-restore-prometheus/).

**Issue:**
- Snapshot data is not showing up in the new Prometheus instance.

**Steps Taken:**
1. Created a snapshot from an older Kubernetes instance containing `chunks/`, `meta.json`, `tombstones`, `index`.
2. Enabled admin API in the new instance.
3. Replaced the snapshot contents in the same directory of the new instance.

I am seeking suggestions on how to properly restore the TSDB. Any insights or guidance would be greatly appreciated.

**Comment by zzen** [comment]
I ran into the same problem, figured it out, forgot the solution, ran into it again and ended up here... Since it seems to catch me by surprise every time, I'll leave a not for my future self here, despite the topic being almost 2 years old:

- the snapshot is just hard links into the main prometheus folder
- the intended use is to copy the contents of `prometheus/snapshorts/<id>/*` into `prometheus/*` on the new machine
- just synchronizing the contents of `snapshot` folders isn't going to work
